internal/resources/k6/resource_project_limits.go

```diff
@@ -151,15 +228,25 @@ func (r *projectLimitsResource) Read(ctx context.Context, req resource.ReadReque
 	// Set the ID to match the project_id
 	state.ID = state.ProjectID
 
+	intID, err := strconv.ParseInt(state.ID.ValueString(), 10, 32)
```

Comment:
There is no "crossplane check" for ID being empty in this resource, it seems. Why?

Reply:
:thinking: so it's a characteristic of the API; I didn't realize that. OK then, thanks for clarifying.